src/app/common/validators/issuer-name.validator.ts

```diff
@@ -0,0 +1,17 @@
+import { AbstractControl } from '@angular/forms';
+
+export class IssuerNameValidator {
```

Comment:
Ok but now you are really mixing together very different tickets, aren't you? I think it would be nice to try to keep PRs as clean as possible. Maybe this is something you could talk about in a meeting, but I think ideally a PR should only solve exactly one ticket. If I see it correctly this PR now covers three tickets and I lost the overview which change belongs to which ticket and what should be solved now.